STP/Tappable.swift

```diff
@@ -25,16 +25,10 @@ public extension Tappable where Self:UIView {
             let tap = recognizer as! UILongPressGestureRecognizer
 
             func isInFrame() -> Bool {
-                let location = recognizer.locationInView(self)
-                switch location {
-                case _ where location.x < 0 - self.frame.size.width / 2,
-                    _ where location.x > self.frame.size.width * 1.5,
-                    _ where location.y < 0 - self.frame.size.height / 2,
-                    _ where location.y > self.frame.size.height * 1.5:
-                    return false
-                default:
-                    return true
-                }
+                return CGRectContainsPoint(
+                  CGRectInset(self.frame, self.frame.width * -0.5, self.frame.height * -0.5),
+                  recognizer.locationInView(self)
+                )
```

Comment:
Hey, no problem.

The `CGRectInset` here returns a `CGRect` thats 50% bigger than `self.frame`. This creates a behavior that allows the gesture to move as far as 50% away from the tappable frame and still making the gesture enabled (the tap still has to be started from within `self.frame` though).

I'm trying to mimmic the behavior of `TouchUpInside`, though I think that API uses a frame 100% larger than the original. (In other words, if the frame is a 50x50y square, the gesture can be moved as far as 50 pixels outside of that square and still engage the tap. But my implementation only allows 25 pixels outside of that square.)

The original `switch` logic did the same thing, but your idea of using `CGRectContainsPoint` is definitely much cleaner. The only strange looking bit is that the `dx` and `dy` parameters in `CGRectInset` needs to be a negative number to return a `CGRect` larger than the original.